File: 2194-cells-in-a-range-on-an-excel-sheet/2194-cells-in-a-range-on-an-excel-sheet.java
class Solution {
    public List<String> cellsInRange(String s) {
        List<String> ans = new ArrayList<>();

        char c1 = s.charAt(0);
        int n1 = (int) s.charAt(1) - '0';
        char c2 = s.charAt(3);
        int n2 = (int) s.charAt(4) - '0';

        for(char ch = c1;ch <= c2; ch++){
            for(int i = n1;i <= n2;i++){
                String t = "" + ch + i;

                ans.add(t);
            }
        }

        return ans;
    }
}

File: 2194-cells-in-a-range-on-an-excel-sheet/2194-cells-in-a-range-on-an-excel-sheet.cpp
class Solution {
public:
    vector<string> cellsInRange(string s) {
        vector<string> ans;

        char c1 = s[0];
        int n1 = (int) s[1] - '0';
        char c2 = s[3];
        int n2 = (int) s[4] - '0';

        for(char ch = c1;ch <= c2; ch++){
            for(int i = n1;i <= n2;i++){
                string t = "";
                t.push_back(ch);
                t.push_back(i+'0');

                ans.push_back(t);
            }
        }

        return ans;
    }
};